### ISSUE TITLE
eval - ConvLLaVA-pretrain-768

### ISSUE THREAD

**Issue by Han-jiaxin** [alibaba/conv-llava#10]
你好，当我测评自己训练好的模型stage3时，测试没有任何问题，当我测试预训练模型ConvLLaVA-pretrain-768时，报错：
- This IS expected if you are initializing LlavaLlamaForCausalLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing LlavaLlamaForCausalLM from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model)

**Comment by John-Ge** [comment]
You should only evaluate the model trained with 3 stages.

**Comment by Han-jiaxin** [comment]
您好，我测试的结果和论文有一些差距，观察发现，第三阶段的训练loss没
![20240829-145627](https://github.com/user-attachments/assets/c9554f03-87f5-4f0b-8ada-4670f27aa94c)
怎么下降，是这个问题导致的效果变差吗？

**Comment by Han-jiaxin** [comment]
您好，我测试的结果和论文有一些差距，观察发现，第三阶段的训练loss没怎么下降，是这个问题导致的性能降低吗？
![20240829-145627](https://github.com/user-attachments/assets/7df750c9-cbbd-4539-a9ad-1487799de0f3)

**Comment by John-Ge** [comment]
Maybe, after the third stage training, training loss should descend to ~ 0.4. Do you use the scripts in the repo?

**Comment by Han-jiaxin** [comment]
是的，我的loss最后一直在0.7~0.9左右，
![20240829-150828](https://github.com/user-attachments/assets/f902a259-2c1c-4861-94c9-2f855496dd22)

**Comment by Han-jiaxin** [comment]
请问咱们有相关的微信或者qq交流群吗，如果有的话，辛苦您拉我一下。

**Comment by John-Ge** [comment]
I have sent you an email.
